mk/spksrc.cross-cmake-env.mk

```diff
@@ -0,0 +1,48 @@
+# Configuration for cmake build
+#
+CMAKE_ARGS = -DCMAKE_INSTALL_PREFIX=$(INSTALL_PREFIX)
+CMAKE_ARGS += -DCMAKE_CROSSCOMPILING=TRUE
+CMAKE_ARGS += -DCMAKE_SYSTEM_NAME=Linux
+CMAKE_ARGS += -DCMAKE_C_COMPILER=$(TC_PATH)$(TC_PREFIX)gcc
+CMAKE_ARGS += -DCMAKE_CXX_COMPILER=$(TC_PATH)$(TC_PREFIX)g++
+CMAKE_ARGS += -DCMAKE_INSTALL_RPATH=$(INSTALL_PREFIX)/lib
+CMAKE_ARGS += -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=TRUE
+CMAKE_ARGS += -DCMAKE_BUILD_WITH_INSTALL_RPATH=TRUE
+
+# set default build directory
+ifeq ($(strip $(PKG_WORK_DIR)),)
+PKG_WORK_DIR = $(WORK_DIR)/$(PKG_DIR)/build
```

Comment:
Not sure I fully get this one?
I guess you referring to libaom special case.
`libaom` could have used `$(WORK_DIR)/$(PKG_DIR)-build` and worked out.  It just could not work under `$(WORK_DIR)/$(PKG_DIR)` thus neither the usual cmake `/build` subdirectory.

In this case I had just decided to reused a "real" versioning directory folder instead of the git hash.  Also as it stands `$(PKG_NAME)_$(PKG_VERS)` will never be null and always use `-build` folder at same level instead.

No?